metadata-ingestion/src/datahub/ingestion/source/powerbi/rest_api_wrapper/data_resolver.py

```diff
@@ -503,13 +503,14 @@ class AdminAPIResolver(DataResolverBase):
         Constant.SCAN_CREATE: "{POWERBI_ADMIN_BASE_URL}/workspaces/getInfo",
         Constant.ENTITY_USER_LIST: "{POWERBI_ADMIN_BASE_URL}/{ENTITY}/{ENTITY_ID}/users",
         Constant.DATASET_LIST: "{POWERBI_ADMIN_BASE_URL}/groups/{WORKSPACE_ID}/datasets",
+        Constant.WORKSPACE_MODIFIED_LIST: "{POWERBI_ADMIN_BASE_URL}/workspaces/modified",
     }
 
-    def create_scan_job(self, workspace_id: str) -> str:
+    def create_scan_job(self, workspace_ids: List[str]) -> str:
```

Comment:
This function wrap the ids in a list eventually and send to PBI eitherway.
So it doesn't matter if u pass in 1 workspace_id or a List[workspace_id], in the end it will still wrap in a List and send.
Actually i recommend sending in a list of workspace to the scan api. Since PBI accept a list, they do it quite quickly too.

But if u want to think async. 
Lets make batch_size configurable then. 

https://github.com/aezomz/datahub/blob/powerbi-improvements/metadata-ingestion/src/datahub/ingestion/source/powerbi/powerbi.py#L1185